lib/msal-browser/docs/acquire-token.md

```diff
@@ -89,10 +79,42 @@ var graphEndpoint = "https://graph.microsoft.com/v1.0/me";
 
 fetch(graphEndpoint, options)
     .then(resp => {
-            //do something with response
+        //do something with response
     });
 ```
 
+## MSAL token acquisition best practices
+
+The following are best practices to acquire tokens with MSAL for avoiding errors, performance hits and usability issues. Certain scenarios may provide exceptions to these.
+
+### Use a single PublicClientApplication instance
+
+Instantiate one `PublicClientApplication` per application and use the same instance throughout your app. This ensures that there is a single source of truth for what MSAL is performing at any given time (see: [MSAL events](events.md)) and eliminates the chance of distinct app objects making parallel interactive requests or potential cache conflicts, which might break apps, reduce performance or hinder user experience.
+
+### Always wait for promises to resolve
+
+All MSAL `acquireToken*` as well as `login*` APIs (which uses `acquireToken*` underneath) perform asynchronous operations and return promises. You should always wait for these promises to resolve, even if you do not need the promise payload.
+
+### Check for interaction status before interactive requests
+
+You should always check for [InteractionStatus](https://azuread.github.io/microsoft-authentication-library-for-js/ref/enums/_azure_msal_browser.interactionstatus.html) being equal to **None** before making an interactive request. `InteractionStatus` will be emitted through [MsalBroadcastService $inProgress observable](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/events.md#the-inprogress-observable) in MSAL Angular and [MsalProvider useMsal() hook](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/docs/hooks.md#usemsal-hook) in MSAL React.
+
+In case of concurrent interactive requests, only the first one will initiate an interaction, while the rest will fail with [interaction_in_progress](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/errors.md#interaction_in_progress) error.
+
+### Attempt silent request first, then interactive
+
+When requesting tokens, always use `acquireTokenSilent` first, falling back to interactive token acquisition if needed. Exception to this is when doing token acquisition outside of an app context that does not have access to interaction status or cannot ensure other components making interactive requests. For instance, if you are making use of `acquireTokenSilent` in a utility function in a React application outside of **MsalProvider context**, you shouldn't attempt to handle *InteractionRequiredAuthError* with an interactive request, but instead route that error to a component that can has access to MSAL context (see: [What can I do outside of msal-react context?](../../msal-react/FAQ.md#what-can-i-do-outside-of-azuremsal-react-context)).
```

Comment:
Same thing here - let's keep this in the React docs

Reply:
done -removed this here